docs/Installation.md

```diff
@@ -41,34 +40,55 @@ minutes to complete.
 Building and flashing the micro-controller
 ==========================================
 
+Via Octoprint, make sure that there is no active serial connection to the printer.
+Fom the OctoPrint web page, under the "Connection" section, click "Disconnect"
+
 To compile the micro-controller code, start by running these commands
-on the Raspberry Pi:
+on the Raspberry Pi via an SSH command line session:
```

Comment:
SSH is not required for this setup process

Reply:
Exactly how many times is the word FTP used in this document vs the word SSH?

The prep part tells you to use SSH to setup Octoprint. They even limited it to SSH because of the security backlash last year. 

Prepping an OS image
Start by installing OctoPi on the Raspberry Pi computer. Use OctoPi v0.14.0 or later - see the octopi releases for release information. One should verify that OctoPi boots and that the OctoPrint web server works. After connecting to the OctoPrint web page, follow the prompt to upgrade OctoPrint to v1.3.7 or later.

**"After installing OctoPi and upgrading OctoPrint, it will be necessary to ssh into the target machine to run a handful of system commands.** If using a Linux or MacOS desktop, then the "ssh" software should already be installed on the desktop. **There are free ssh clients** available for other desktops (eg, PuTTY). Use the ssh utility to connect to the Raspberry Pi (ssh pi@octopi -- password is "raspberry") and run the following **commands:**"
 
Download PuTTY - a free SSH and telnet client for Windows

Octoprint also tells you to do the exact same thing.
https://octoprint.org/download/

Reply:
Ok, I stand corrected. I installed mine without a network connection, with my pi connected to a screen and a keyboard. But if that's what Octoprint and klipper recommend in the rest of the documentation, you're right.